File: go-sdk/pkg/models/operations/unpublishuser.go
package operations

type UnpublishUserPathParams struct {
	ID int32 `pathParam:"style=simple,explode=false,name=id"`
}

type UnpublishUserRequest struct {
	PathParams UnpublishUserPathParams
}

type UnpublishUserResponse struct {
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/unpublishuser.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class UnpublishUserPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class UnpublishUserRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: UnpublishUserPathParams;
}


export class UnpublishUserResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
